modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -3,35 +3,51 @@
 
-This page describes managing database aliases for standard databases.
-Local and remote database aliases can also be created as part of a xref:database-administration/composite-databases/concepts.adoc[composite database].
-For more information, see xref:database-administration/aliases/manage-aliases-composite-databases.adoc[].
+[NOTE]
+====
+If a transaction modifies a database alias, other transactions concurrently executing against that alias may be aborted and rolled back for safety.
+This prevents issues such as a transaction executing against multiple target databases for the same alias.
+====
+
+There are two kinds of database aliases - local and remote:
 
-A local database alias can be used in all other Cypher commands in place of the target database.
+Local database aliases::
+A local database alias can only target a database within the same DBMS.
+It can be used in all Cypher commands in place of the target database.
 Please note that the local database alias will be resolved while executing the command.
-Privileges are defined on the database, and not the local database alias.
+Privileges are defined on the target database, and not the local database alias.
+
+
+Remote database aliases::
+A remote database alias may target a database from another Neo4j DBMS.
+It can be used for:
+* Connecting to a database of a remote Neo4j DBMS
+* `USE` clauses
+* Setting a user's home database
+* Defining the access privileges to the remote database
+* label:new[Introduced in 2025.06] Setting a default Cypher version for queries to the remote database.
 
-A remote database alias can be used for connecting to a database of a remote Neo4j DBMS, `USE` clauses, setting a user's home database, and defining the access privileges to the remote database.
 Remote database aliases require configuration to safely connect to the remote target, which is described in xref::database-administration/aliases/remote-database-alias-configuration.adoc[Connecting remote databases].
 It is not possible to impersonate a user on the remote database or to execute an administration command on the remote database via a remote database alias.
 
-Database aliases can be created and managed using a set of Cypher administration commands executed against the `system` database.
-The required privileges are described in the xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[The DBMS ALIAS MANAGEMENT privileges].
-When connected to the DBMS over Bolt, administration commands are automatically routed to the `system` database.
-
 [NOTE]
 ====
-If a transaction modifies a database alias, other transactions concurrently executing against that alias may be aborted and rolled back for safety.
-This prevents issues such as a transaction executing against multiple target databases for the same alias.
+Starting with Neo4j 2025.06, a database or remote alias can be assigned a default Cypher version.
```

Comment:
And with the fix for https://github.com/neo4j/docs-operations/pull/2372#discussion_r2175050577 we're back to my initial point of https://github.com/neo4j/docs-operations/pull/2372#discussion_r2169122575:

wouldn't it make more sense to have this note just below the list talking about default Cypher version instead of having it after the sentence about connecting?

The sentence about connecting isn't at all indented anymore so it makes more sense to have the two things related to default language together
![image](https://github.com/user-attachments/assets/3fa073af-c0e3-47f1-9543-366c47aa1b64)

Reply:
So, if I add right after the default Cypher bullet, it will be part of the bullet. Probably, I need to move it just below Connecting to a database of a remote Neo4j DBMS and leave the note alone:
<img width="807" alt="Screenshot 2025-07-01 at 11 45 20" src="https://github.com/user-attachments/assets/5d741f40-c316-4c78-a817-c2b960b9cabe" />